src/components/structures/VectorHomePage.js

```diff
@@ -24,7 +24,10 @@ import sanitizeHtml from 'sanitize-html';
 import sdk from 'matrix-react-sdk/lib';
 
 module.exports = React.createClass({
-    displayName: 'HomePage',
+    displayName: 'VectorHomePage',
+    statics: {
+        replaces: 'HomePage',
```

Comment:
This looks like it needs a comment.

Reply:
i forgot about this. I need to turn HomePage into a ES6 class and then have VectorHomePage inherit from it (whilst also `replace`ing it)